bin/check-redis-list-length.rb

```diff
@@ -63,21 +63,45 @@ class RedisListLengthCheck < Sensu::Plugin::Check::CLI
          description: 'Redis list KEY to check',
          required: true
 
+  banner "Usage: #{File.basename($PROGRAM_NAME)} (options)\n" \
+    'Number of items in list key will have to be below threshold if warning is smaller than critical, and above otherwise'
+
   def run
     options = { host: config[:host], port: config[:port], db: config[:database] }
     options[:password] = config[:password] if config[:password]
-    redis = Redis.new(options)
 
+    check_length_is_above_threshold = false
+    if config[:warn] > config[:crit]
```

Comment:
Also this makes this a breaking change. I am gonna actually reverse my stance and say we should switch back to adding a parameter that is the conditional so it could handle: `>=`, `>`, `==`, `<=`, `<` without a breaking change and becomes more obvious to its use without inspection of the code.